lib/spack/spack/build_systems/intel.py

```diff
@@ -103,6 +136,742 @@ def _filtered_components(self):
 
         return matches
 
+    @property
+    def _want_shared(self):
+        return ('+shared' in self.spec)
+
+    @property
+    def intel64_int_suffix(self):
+        '''Provide the suffix for Intel library names to match a client
+        application's int size.
+
+            ilp64: all of int, long, and pointer are 64 bit.
+             lp64: only long and pointer are 64 bit; int will be 32bit.
```

Comment:
This won't format the way you want in the documentation. Might want to use an rST table.